src/pages/WatchList.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import ManageBtn from '../components/UI/ManageList/ManageBtn'
+import WatchListItem from '../components/WatchListItem'
+import { IStoreItem, WatchListEnum } from '../models'
+
+const WatchList = () => {
+
+  const watchList = useSelector((state: IStoreItem) => state.watchList)
+  const dispatch = useDispatch()
+
+  const listClear = () => {dispatch({
```

Comment:
function name must be a verb
```suggestion
  const clearList = () => {dispatch({
```